### DISCUSSION TITLE
Docs for synth API?

### DISCUSSION THREAD

**Discussion by free-variation** [asb2m10/dexed#360] [Q&A]
Greetings, and thanks for the excellent Dexed.

Is there any documentation on how to talk to the synth engine programmatically (in whatever programming language)?  I'd like to be able to script things in code, totally bypassing the UI.

Thanks again.

**Comment by frederico-betting** [comment]
Do you have any update on this @free-variation ? I also would like to have access to this kind of documentation…

**Reply by free-variation** [reply]
Unfortunately not :(
